New behavior for container message extractor is not opt-in (#908)
Particular/NServiceBus.Persistence.CosmosDB: src/NServiceBus.Persistence.CosmosDB.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

        public static NServiceBus.PersistenceExtensions<NServiceBus.CosmosPersistence> DatabaseName(this NServiceBus.PersistenceExtensions<NServiceBus.CosmosPersistence> persistenceExtensions, string databaseName) { }
        public static NServiceBus.PersistenceExtensions<NServiceBus.CosmosPersistence> DefaultContainer(this NServiceBus.PersistenceExtensions<NServiceBus.CosmosPersistence> persistenceExtensions, string containerName, string partitionKeyPath) { }
        public static void DisableContainerCreation(this NServiceBus.PersistenceExtensions<NServiceBus.CosmosPersistence> persistenceExtensions) { }

>         [System.Obsolete("Using extracted container information from the incoming message will become the d" +
>             "efault behavior starting in version 4.0, making this API redundant. Will be trea" +
>             "ted as an error from version 4.0.0. Will be removed in version 5.0.0.", false)]

Reviewer: ```suggestion
        [System.Obsolete("When a default container and a message container extractor is configured, " + 
        "using the message container extractor will become the default behavior starting in version 4.0, " +
        "making this API redundant. Will be treated as an error from version 4.0.0. Will be removed " +
        "in version 5.0.0.", false)]
```
